### PR TITLE
Adding manifests and code updates for kubernetes deployment

### PR DESCRIPTION
**Pull request by dkirby-ms**
New kubernetes manifests:
- backend_db
- backend_api
- rtsp

Changes to code to support kubernetes deployment

- add docker compose files for store db and api
- add readme with deploy guide for local docker and kubernetes

### CODE REVIEW

File: agora/main_ui/operations/backend_api.yaml [hunk 1]
```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: initdb-configmap
+  namespace: contoso-hypermarket
+data:
+  initdb.sql: |
```
Comment (fcabrera23) [inline agora/main_ui/operations/backend_api.yaml:7]: Why not use the file initdb.sql ?

File: agora/main_ui/operations/pushImages.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (fcabrera23) [inline agora/main_ui/operations/pushImages.sh:1]: I think we dont need this - We will have a pipeline that builds and pushes everything to MCR

File: agora/README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
-## Contoso Hypermarket
+# Contoso Hypermarket
```
Comment (fcabrera23) [inline agora/README.md:1]: Maybe we could have a folder named contoso-hypermarket that we can have the instrucitons and docker compose?

**Review by fcabrera23 — CHANGES_REQUESTED** [review]

File: agora/main_ui/operations/pushImages.sh (hunk 2 above)
Comment (dkirby-ms) [reply agora/main_ui/operations/pushImages.sh:1]: We dont need it in the final version, but its useful for dev. I can just keep it locally in my machine and remove here.

File: agora/main_ui/operations/backend_api.yaml (hunk 1 above)
Comment (dkirby-ms) [reply agora/main_ui/operations/backend_api.yaml:7]: Good question, i couldnt get it working quickly and wanted to get this out the door today. I will revisit.

File: agora/README.md (hunk 3 above)
Comment (dkirby-ms) [reply agora/README.md:1]: I will move them.

File: agora/main_ui/operations/backend_api.yaml (hunk 1 above)
Comment (dkirby-ms) [reply agora/main_ui/operations/backend_api.yaml]: this is addressed now and inline SQL removed

**Pull request closed without merging**